docs/references/general/dat.md

```diff
@@ -0,0 +1,113 @@
+| *hour*                 | *hrs*     | short     | Hour                                                                                                                                                                                             |
+| *minut*                | *min*     | short     | Minute                                                                                                                                                                                           |
+| *sec*                  | *s*       | short     | Seconds                                                                                                                                                                                          |
+| *txpow*                | *kW*      | short     | Transmitted power                                                                                                                                                                                |
+| *nave*                 | *None*    | short     | Number of pulse sequences transmitted                                                                                                                                                            |
+| *atten*                | *None*    | short     | Attenuation level                                                                                                                                                                                |
+| *lagfr*                | *us*      | short     | Lag to first range                                                                                                                                                                               |
+| *smsep*                | *us*      | short     | Sample separation                                                                                                                                                                                |
+| *ercod*                | *None*    | short     | Error Code                                                                                                                                                                                       |
+| *stat.agc*             | *None*    | short     | Automatic gain control status word                                                                                                                                                               |
+| *stat.lopwr*           | *None*    | short     | Low power status word                                                                                                                                                                            |
+| *nbaud*                | *None*    | short     | The number of elements in the pulse code                                                                                                                                                         |
```

Comment:
I believe this is a  parameter in dat files but old raw actually doesn't read or write this value? 
Maybe we should remove it?